### ISSUE TITLE
--tag appears to be backwards

### ISSUE THREAD

**Issue by sazzer** [bbqsrc/cucumber-rust#46]
When I'm running tests, I've noticed that there is an `--tag` command line option with the help text:
```
    -t, --tag <tag>             Filter by specified tag
```

However, when I use this it appears to run the tests *without* this tag, instead of the ones *with* it.

Specifically, I use the tag `@wip` for the tests I'm currently working on, and running both `target/debug/deps/cucumber-295c8d12068f02de --tag wip` and `target/debug/deps/cucumber-295c8d12068f02de --tag @wip` is running all of the tests that do *not* have `@wip` associated with them.

**Comment by bbqsrc** [comment]
Hum. Bizarre. Will have a look.

**Comment by bbqsrc** [comment]
Point your dep to `git = "https://github.com/bbqsrc/cucumber-rust"` and let me know if I fixed it. You need to use `@wip`, `wip` won't be picked up.

**Comment by sazzer** [comment]
Afraid not.

Using `--tag @wip` now runs everything, and using `--tag wip` still runs everything that does *not* have `@wip` on it.

**Comment by sazzer** [comment]
That's with my `Cargo.toml` file saying:
```
cucumber_rust = { git = "https://github.com/bbqsrc/cucumber-rust" }
```

**Comment by bbqsrc** [comment]
Haha, what. Can you show me your feature file?

**Comment by sazzer** [comment]
Sorry - Ignore that!

Turns out my machine had gone wrong and wasn't using it after all. I've just restarted it and it *is* working correctly now :)

**Comment by bbqsrc** [comment]
Haha, I'm glad. I was questioning my sanity a bit there, given there's basically just `!tags.contains(tag)`. I'll drop a patch release in a little bit.

**Comment by bbqsrc** [comment]
Fixed in v0.6.8. 😄
